docs/css/README.md

```diff
@@ -79,10 +181,10 @@ We organize our partials into separates folders:
 | settings   | Variables & common settings                                                                                           | Usually one `_settings` file                                      |
 | tools      | Sass functions and mixins                                                                                             | Usually one `_mixins` and one `_functions` file                   |
 | vendor     | Third-parties code, prefer Node modules when available                                                                |                                                                   |
-| defaults   | All the defaults & global elements (html, body, h1, p, blockquote, …)                                                 | We often see `_layout`, `_typography`, `_forms`, or `_fonts` here |
+| base       | All the default styling on global elements (html, body, h1, p, blockquote, …)                                                 | We often see `_layout`, `_typography`, `_forms`, or `_fonts` here |
```

Comment:
More than `global html elements` it can also be "logical components" such as `navbar` or `text-field`.